### ISSUE TITLE
Option to send airspeed instead of gps speed for CRSF radios

### ISSUE THREAD

**Issue by juanxli** [iNavFlight/inav#9937]
## Current Behavior
There isn't an option for this and you are stuck with GPS speed on CRSF/ELRS on the radio screen 

## Desired Behavior

**Being able to change the source of the speed shown on CRSF radios ** between airspeed and GPS speed.

## Suggested Solution

Add a function that substitutes the airspeed instead of the GPS speed telemetry of the radio.
An option similar to  "osd_speed_source" but instead "telemetry_speed_source"

## Who does this impact? Who is this for?

Wings with CRSF/ELRS and a pitot tube/airspeed sensor without an FPV system 

## Additional context

This is because FPV isn't allowed in some clubs or legislations only Line of sight

**Comment by SpidaTuSefen** [comment]
Frsky have this and lots of LOS pilots use it so their airspeed is called out on final approach when landing bigger, scale fixed wing.

**Comment by SpidaTuSefen** [comment]
It would also mimic airspeed calls by co-pilot used in manned aircraft - very professional way to handle take off, climb and landing.

**Comment by SpidaTuSefen** [comment]
FPV is only legal at MAAA clubs in AU which a few and far apart, not to individual pilots at public spaces. So LOS iNav is the only legal option. C Asa even consider a monitor to be FPV.

**Comment by ShikOfTheRa** [comment]
This would be really usefull. I am surprised no more upvotes.
Flying with a glider and need to keep airspeed above stall speed. Cant use alarms on tx with gps speed because sometimes airspeed = 3-kmh and gps speed = 0kmh...

Possible to add it as cli feature? replace gps speed with air speed when sending over crsf?

**Comment by dcan999** [comment]
Did this go anywhere? Didn't see anything in 8.x release but would really like it. When I lose video I like to monitor the RTH using the radio telemetry and would like to see the airspeed rather than ground speed to know if I need to override the throttle etc.

**Comment by viennalex76-alt** [comment]
ELRS protocol since version 3.5.5 now supports telemetry for airspeed sensor via CRSF. Would be fine to use this in iNav, so no need to switch between GPS-Speed and Airspeed

**Comment by JetsRfaster** [comment]
@ viennalex76-alt Upgraded to 3.6.2 but the digital Airspeed sensor is not showing up in the EdgeTx sensor list?  I'm using a Jumper T15 radio with the latest EdgeTx OS.  24 items are listed under Telemetry sensors, and Airspeed is working on the OSD, but Airspeed not listed as a radio ELRS sensor?  Any suggestions?

**Comment by viennalex76-alt** [comment]
@JetsRfaster You're right and your issue is absolutely relatable because iNav itself doesn't send any Airspeed data via CRSF, that's why you don't see any airspeed data on EdgeTX. The ELRS protocole supports the Airspeed data in a dedicated frame, but iNav needs to be adapted to be able to literally send the Airspeed data into the CRSF protocol... It needs to create a new frame (0A to be precise) containing the airspeed value and add this into the telemetry queue. This is exactly what the issue #11025 tries to create. Unfortunately it doesn't work that well and many bugs still need to be fixed.

**Comment by JetsRfaster** [comment]
@viennalex76-alt noted thanks. Please keep us posted if the airspeed code is solved. 🙏

**Comment by sensei-hacker** [comment]
Someone tried to add airspeed in https://github.com/iNavFlight/inav/pull/11025 , but after I merged it we found out it broke other things. So I had to revert it.

If anyone wants to try to debug it, that would be cool.
